.github/jobs/ios.yml

```diff
@@ -28,10 +28,12 @@ jobs:
 
     - task: Xcode@5
       inputs:
+        actions: 'build'
         xcWorkspacePath: 'buildiOS/BabylonNative.xcodeproj'
         scheme: 'Playground'
         sdk: 'iphoneos'
         useXcpretty: false
         configuration: RelWithDebInfo
+        destinationTypeOption: simulators
```

Comment:
Just curious, what does this mean? It seems like it conflicts with `sdk: 'iphoneos'`.

Reply:
leftover for some tests. removed.